src/views/EventsAttended/index.js

```diff
@@ -33,47 +33,9 @@ const EventsAttended = (props) => {
     },
   };
 
-    if (loading === true) {
-      content = <GordonLoader />;
-    } else if (props.authentication){
-    } else if (events.length > 0) {
-      content = (
-        <Grid container direction="row" justify="center" spacing="2">
-          <Grid item align="center">
-            <Button
-              variant="contained"
-              style={style.button}
-              component={Link}
-              to="/events?CLW%20Credits"
-            >
-              Need More Chapel Credits?
-            </Button>
-          </Grid>
-          <Grid item>
-            <EventList events={events} />
-          </Grid>
-        </Grid>
-      );
-    } else {
-      content = (
-        <Grid item align="center">
-          <br /> <br />
-          <Typography variant="h4" align="center">
-            No Events To Show
-          </Typography>
-          <br />
-          <Button
-            variant="contained"
-            style={style.button}
-            component={Link}
-            to="/events?CLW%20Credits"
-          >
-            Need More Chapel Credits?
-          </Button>
-        </Grid>
-      );
-    }
-  } else {
+  if (loading === true) {
+    content = <GordonLoader />;
+  } else if (!props.authentication) {
```

Comment:
I did not incorporate the exact change that Evan mentioned as I wasn't sure exactly he was referring too, but this should clear up some of the logic for this section. Let me know if it looks good!